mozilla_repo_urls/parser.py

```diff
@@ -9,7 +9,7 @@
     giturlparse.platforms.PLATFORMS.insert(i, platform)
 
 
-_SUPPORTED_PLAFORMS = ("hgmo", "github")
+_SUPPORTED_PLATFORMS = ("hgmo", "github")
 
```

Comment:
Refactor: named module constant.

```suggestion
_SUPPORTED_HOSTS = tuple(sorted([
    host
    for domains in [
        platform[1].DOMAINS
        for platform in giturlparse.platforms.PLATFORMS
        if platform[0] in _SUPPORTED_PLATFORMS
    ]
    for host in domains
]))

```

We can make it a sorted tuple too, which should simplify testing. Speaking of testing, we can create a new file at `test/test_parser.py` with this content:

```
from mozilla_repo_urls import parser


def test_supported_hosts():
    assert parser._SUPPORTED_HOSTS == ("gist.github.com", "github.com", "hg.mozilla.org")
```

This way, it's simpler to see what hosts we actually support. How does that sound?